### DISCUSSION TITLE
iSH on iPad 2 (iOS 9)

### DISCUSSION THREAD

**Discussion by ifrh** [ish-app/ish#1283] [Q&A] [answered]
Hi there,
does any one know how to install iSH on iPad2 running iOS 9?
I just want some way to run npm and project ( https://github.com/SuperMarcus/airprint-proxy ) directly on that old iPad.
And same for an iPhone X (running iOS 14).

Best regards.

**Comment by tbodt** [comment, marked as answer]
iSH requires iOS 11, so you can't install it on the iPad 2. You should be able to install it on your iPhone X from the app store, though.
